xml/System.Runtime.InteropServices/LibraryImportAttribute.xml

```diff
@@ -20,8 +20,18 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Attribute used to indicate a source generator should create a function for marshalling arguments instead of relying on the runtime to generate an equivalent marshalling function at run-time.</summary>
```

Comment:
```suggestion
    <summary>Indicates that a source generator should create a function for marshalling arguments instead of relying on the runtime to generate an equivalent marshalling function at run time.</summary>
```